### DISCUSSION TITLE
Add custom text/banner area to daily challenge screen

### DISCUSSION THREAD

**Discussion by spacekotek** [ppy/osu#31078] [Ideas]
The idea is to add custom text to the daily challenge screen that can display stuff like "some artist week" or "some genre week" or just some fitting text for the beatmap
![изображение](https://github.com/user-attachments/assets/fa9c8e73-a9f2-4afa-818c-84faf68dbe77)
![изображение](https://github.com/user-attachments/assets/4e6d3e2e-055d-480e-9f73-72b8ab65377e)
![изображение](https://github.com/user-attachments/assets/ef94d00b-f16d-447a-96d1-915862bf0590)

**Comment by Gabixel** [comment]
A niche addition, but seems like a fun little thing to have (some kind of [MOTD](https://en.wikipedia.org/wiki/Message_of_the_day) for the lazer Daily Challenge, maybe not necessarily always closely related to the current beatmap), maybe even in the main menu somehow.

**Comment by peppy** [comment]
Not against this. Could also be done in the form of a custom image.
